src/main/webapp/app/exam/participate/exam-participation.component.ts

```diff
@@ -27,6 +30,8 @@ import { throttleTime } from 'rxjs/operators';
 import * as moment from 'moment';
 import { Moment } from 'moment';
 
+type GenerateParticipationStatus = 'generating' | 'failed' | 'success';
```

Comment:
:+1:

Reply:
not entirely sure if you mean that ironic :)

Reply:
Nah, I like those 🙂